workshops/node_intro/README.md

```diff
@@ -0,0 +1,288 @@
+---
+name: Node Intro + Express
+description: Make a simple webserver!
+author: 'Cedric Hutchings'
+group: start
+order: 9
+---
+
+# Express Webserver
+
+## What can you learn in this workshop?
+A webserver is a little bit of software that lets you send websites -- and all of the files that they need -- to anyone who connects to your server.
+I'll admit, making a webserver seems a little prosaic.
+It's not even immediately evident *why* you'd want to bother making one.
+In truth, I'm making this workshop as the beginning of a tutorial series I'm running on how to make a webgame with networked multiplayer, but there are plenty of other reasons to make your own webserver.
+
+First of all, I think you should do this workshop just to get more familiar with navigating through your own filesystem using the **command line**, if you aren't already.
+I introduce a few commands as I go through, and once you have these you're only a couple extra googles away from becoming fairly proficient at using the command line.
+That can make you a good bit faster when manipulating the filesystem, especially if you're already a fast typist.
+
+**Node.js** is a pretty cool technology for doing lots of things, especially developing web related software. This project is an awesome introduction to node
+because it walks you through configuring a brand new node project.
+Surely that could come in handy at a Hackathon some time in the future, when you want to use a library for Machine Learning or a library for some other fancy concept that uses Node.
+
+**Express** is awesome for quickly hosting files from your own computer.
+There are a bunch of other ways to host files from your computer, including some command line interfaces that don't require any code at all,
+but because Express is just Node.js code, it offers a lot of opportunities for doing some fairly complex configuration and web app serving.
+Perhaps it's overkill for a webgame which has fairly simple webserver needs, but I like to keep my options open in case I want to add some pages to my app for making an account or looking at your player stats and whatnot.
+
+`Repl.it`/`Glitch`/`Github Pages` do offer their own interfaces you can use to get your files hosted, but here are a few reasons why you'd want to write your own with Express:
```

Comment:
Change to:
`Repl.it`, `Glitch`, and `Github Pages` do offer their own file hosting services, but here are a few reasons why you'd want to write your own with Express: